server/utils/forms/inputs/meetingMethodInput.ts

```diff
@@ -0,0 +1,28 @@
+import { Request } from 'express'
+import { FormValidationResult } from '../formValidationResult'
+import { AppointmentDeliveryType } from '../../../models/actionPlan'
+
+export default class MeetingMethodInput {
```

Comment:
Would be good to add a test for this class (unless I've missed something!)

Reply:
Yes, I've added this now (Sorry it was in a draft PR accidentally)